transient-aws/hive-example/cluster_bare_amis.conf

```diff
@@ -0,0 +1,194 @@
+
+#
+# Copyright (c) 2016 Cloudera, Inc. All rights reserved.
+#
+
+#
+# Simple AWS Cloudera Director configuration file with automatic role assignments
+# that works as expected if you use a single instance type for all cluster nodes
+#
+
+#
+# Cluster name
+#
+
+name: ETL-Transient
+
+#
+# Cloud provider configuration (credentials, region or zone and optional default image)
+#
+
+provider {
+    type: aws
+
+    accessKeyId: "REPLACE_ME"
+    secretAccessKey: "REPLACE_ME"
+
+    publishAccessKeys: false
+
+    region: us-west-2[REPLACE_ME]
+
+    subnetId: subnet-41b93525[REPLACE_ME]
+
+    securityGroupsIds: sg-0cf7326a [REPLACE_ME]
+
+    instanceNamePrefix: hive-cdh
+
+    rootVolumeSizeGB: 30 # matching the size of the pre-extracted AMI
+    rootVolumeType: gp2 # OR standard (for EBS magnetic)
+
+    associatePublicIpAddresses: true
+}
+
+#
+# SSH credentials to use to connect to the instances
+#
+
+ssh {
+    username: ec2-user [REPLACE_ME]
+    privateKey: """-----BEGIN RSA PRIVATE KEY-----
+        [REPLACE_ME]
+    -----END RSA PRIVATE KEY-----
+"""
+}
+
+#
+# A list of instance types to use for group of nodes or management services
+#
+
+instances {
+    master {
+        type: c4.xlarge
+        image: ami-775e4f16 [REPLACE_ME]
```

Comment:
Meh, I guess it's better not to suggest an AMI. I believe for RHEL they publish new AMIs every few months for an OS version, I suppose with the latest patches. So, any AMI we suggest could become out-of-date and potentially a security problem.